### PR TITLE
implemented the reviews star ratings

### PR DESCRIPTION
**Pull request by Bosris**

### CODE REVIEW

File: client/components/ReviewList.jsx [hunk 1]
```diff
@@ -56,7 +57,10 @@ class ReviewList extends React.Component {
   render() {
     return (
       <div style={styles.container} id='review-list'>
-        <h4>Reviews ***** ({this.props.reviews})</h4>
+        <div style={styles.reviewHeader}>
+          <h4 style={{ display: 'flex' }}> Reviews <span style={{ marginLeft: '5px', marginRight: '5px' }}><StarRatingComponent value={this.props.ratings} starCount={5} starColor={'black'} emptyStarColor={'#E1E3DF'} /> </span>({this.props.reviews})</h4>
```
Comment (silkyh13) [inline client/components/ReviewList.jsx]: Consider separating your line here into multiple lines.

File: client/components/ReviewList.jsx [hunk 2]
```diff
@@ -56,7 +57,16 @@ class ReviewList extends React.Component {
   render() {
     return (
       <div style={styles.container} id='review-list'>
-        <h4>Reviews ***** ({this.props.reviews})</h4>
+        <div style={styles.reviewHeader}>
+          <h4 style={{ display: 'flex' }}>
+            Reviews
+            <span style={{ marginLeft: '5px', marginRight: '5px' }}>
+              <StarRatingComponent value={this.props.ratings} starCount={5} starColor={'black'} emptyStarColor={'#E1E3DF'} />
```
Comment (therobinkim) [inline client/components/ReviewList.jsx]: `starColor={'black'}` => `starColor='black'`
`emptyStarColor={'#E1E3DF'}` => `emptyStarColor='#E1E3DF'`

File: package.json [hunk 3]
```diff
@@ -26,6 +26,7 @@
     "nodemon": "^1.19.3",
```
Comment (therobinkim) [inline package.json]: `nodemon` should be a `devDependency`

File: public/bundle.js [hunk 4]
```diff
@@ -11,20 +11,25 @@ object-assign
...
...nction(e,t,n){return H().useReducer(e,t,n)},useRef:function(e){return H().useRef(e)},useState:function(e){return H().useState(e)},Fragment:o,Profiler:c,StrictMode:u,Suspense:p,unstable_SuspenseList:m,createElement:O,cloneElement:function(e,t,n){if(null==e)throw g(Error(267),e);var l=r({},e.props),i=e.key,o=e.ref,u=e._owner;if(null!=t){if(void 0!==t.ref&&(o=t.ref,u=_.current),void 0!==t.key&&(i=""+t.key),e.type&&e.type.defaultProps)var c=e.type.defaultProps;for(s in t)P.call(t,s)&&!N.hasOwnProperty(s)&&(l[s]=void 0===t[s]&&void 0!==c?c[s]:t[s])}var s=arguments.length-2;if(1===s)l.children=n;else if(1<s){c=Array(s);for(var f=0;f<s;f++)c[f]=arguments[f+2];l.children=c}return{$$typeof:a,type:e.type,key:i,ref:o,props:l,_owner:u}},createFactory:function(e){var t=O.bind(null,e);return t.type=e,t},isValidElement:M,version:"16.10.2",unstable_withSuspenseConfig:function(e,t){var n=C.suspense;C.suspense=void 0===t?null:t;try{e()}finally{C.suspense=n}},__SECRET_INTERNALS_DO_NOT_USE_OR_YOU_WILL_BE_FIRED:{ReactCurrentDispatcher:S,ReactCurrentBatchConfig:C,ReactCurrentOwner:_,IsSomeRendererActing:{current:!1},assign:r}},V={default:W},B=V&&W||V;e.exports=B.default||B},function(e,t,n){"use strict";
```
Comment (therobinkim) [inline public/bundle.js]: You should never commit your `bundle.js`; you should remove it from your repo entirely

**Comment by therobinkim** [comment]
I recommend configuring ESLint for Airbnb's style guide asap so you can benefit from using it for as long as possible.

**Pull request merged**